receiver/awsecscontainermetricsreceiver/awsecscontainermetrics/label_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package awsecscontainermetrics
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestContainerLabelKeysAndValues(t *testing.T) {
+	cm := ContainerMetadata{
+		ContainerName: "container-1",
+		DockerID:      "001",
+		DockerName:    "docker-container-1",
+	}
+	k, v := containerLabelKeysAndValues(cm)
+	require.EqualValues(t, 3, len(k))
```

Comment:
Would having more detailed label content checks be helpful with these cases or will they be prone to change?

Reply:
I think it's fine as we will always have correct values if we come to this point. Also, I made a little change- moved the hardcoded length to a constant file.

Reply:
Additionally, I added some minor checks.